src/components/Tool/Appeal/AppealsContext/AppealsContext.tsx

```diff
@@ -178,9 +178,25 @@ export const AppealsProvider: React.FC<AppealsContextProps> = ({
     },
     skip: contactCount === 0,
   });
+  // In the flows view, we need the total count of all contacts in every column, but the API
+  // filters out contacts excluded from an appeal. We have to load excluded contacts also and
+  // manually merge them with the other contacts.
+  const { data: allExcludedContacts } = useGetIdsForMassSelectionQuery({
```

Comment:
We grab the excluded contacts on line `275`, maybe you could use that query instead?

Reply:
That query just loads the total contact count. We need the contact ids for mass selection.